### PR TITLE
feat: use the official gnosis network icon

### PR DESCRIPTION
**Pull request by firebomb1**
## Description

Use the Official Gnosis logo for asset icons and network icon. xDAI remains the fee currency and keeps its specific logo.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #

## Risk
None.

## Testing
- Check that the assets that are on Gnosis and which have network icons in certain interfaces (Assets page, Assets dropdowns) have the proper Gnosis icon.
- Check that the Gnosis network in the network selection menu with the non-native wallets (e.g. MetaMask) show the proper icon.

### Engineering
☝️ 

Note: There will be some overlap with the changes in #4641 it also seem some of the Cosmos URL for icons get updated after each run of `yarn generate::asset-data`, not sure which ones should be used, they both seem to work, as well as the previous ones.

### Operations
☝️ 

## Screenshots (if applicable)
Before

image
image

After

image
image

### CODE REVIEW

**Comment by woodenfurniture** [comment]
note requires action of https://github.com/shapeshift/web/pull/4641#pullrequestreview-1461419265

File: src/lib/swapper/swappers/utils/test-data/assets.ts [hunk 1]
```diff
@@ -185,6 +185,8 @@ export const XDAI: Asset = {
   precision: 18,
   color: '#33765c',
   icon: 'https://assets.coingecko.com/coins/images/11062/large/Identity-Primary-DarkBG.png?1638372986',
+  networkIcon:
```
Comment (gomesalexandre) [inline src/lib/swapper/swappers/utils/test-data/assets.ts:188]: ditto

**Review by gomesalexandre — COMMENTED** [review]
Very nice catch @firebomb1 ! That's a conceptual stamp, indeed there has been some confusion in XDAI native asset vs. Gnosis the network here, and that's the icon we should show for the network 🤌 

Happy to stamp and merge once its [dependency](https://github.com/shapeshift/web/pull/4641) is

**Comment by firebomb1** [comment]
> Happy to stamp and merge once its dependency is

I'm unsure when this will happen, I'm not comfortable modifying the asset-data generation script for Cosmos/Osmosi as I'm not very accustomed with what our app does on that network.

Maybe I can revert the unrelated changes on `src/lib/asset-service/service/generatedAssetData.json` and only leave the XDAI/Gnosis related one?

**Comment by gomesalexandre** [comment]
> > Happy to stamp and merge once its dependency is
> 
> I'm unsure when this will happen, I'm not comfortable modifying the asset-data generation script for Cosmos/Osmosi as I'm not very accustomed with what our app does on that network.
> 
> Maybe I can revert the unrelated changes on `src/lib/asset-service/service/generatedAssetData.json` and only leave the XDAI/Gnosis related one?

That would work, though we won't see it at runtime yet unfortunately so might as well land it at once

File: scripts/generateAssetData/baseAssets.ts [hunk 2]
```diff
@@ -183,6 +183,8 @@ export const gnosis: Asset = {
   precision: 18,
   color: '#33765c',
   icon: 'https://assets.coingecko.com/coins/images/11062/large/Identity-Primary-DarkBG.png?1638372986',
+  networkIcon:
+    'https://assets.coingecko.com/asset_platforms/images/11062/large/Aatar_green_white.png?1643204471',
```
[the remark this answers is not in the document]
Comment (gomesalexandre) [reply scripts/generateAssetData/baseAssets.ts:187]: Yeah if we consistently have it let's keep it, I doubt the icon will change anytime soon but might as well!

File: scripts/generateAssetData/baseAssets.ts [hunk 3]
```diff
@@ -174,7 +174,7 @@ export const polygon: AssetWithNetworkName = {
   explorerTxLink: 'https://polygonscan.com/tx/',
 }
 
-export const gnosis: Asset = {
+export const gnosis: AssetWithNetworkName = {
```
Comment (gomesalexandre) [inline scripts/generateAssetData/baseAssets.ts:177]: 💜

**Review by gomesalexandre — APPROVED** [review]

File: scripts/generateAssetData/baseAssets.ts (hunk 3 above)
Comment (firebomb1) [reply scripts/generateAssetData/baseAssets.ts:177]: 👍

**Pull request merged**